solana/gatekeeper-lib/src/commands/network-feature.ts

```diff
@@ -62,7 +67,7 @@ export default class AddGatekeeper extends Command {
     const { args, flags } = await this.parse(AddGatekeeper);
 
     const networkFeature: NetworkFeature = args.feature as NetworkFeature;
-    const gatekeeperNetwork = flags.gatekeeperNetworkKey as Keypair;
+    const gatekeeperNetwork = flags.gatekeeperKeypair as Keypair;
```

Comment:
Danger, something is not right here (also previously). Assigning Gatekeeper to Network, which is functionally not correct.